### ISSUE TITLE
nvda doesn't announce volume level outside of explorer when using volume physical buttons

### ISSUE THREAD

**Issue by mohammad-aloufi** [nvaccess/nvda#14369]
### Steps to reproduce:
Raise or lower the volume using the volume physical buttons when you're not focused on a windows explorer window.
### Actual behavior:
nvda doesn't announce the new volume level
### Expected behavior:
nvda should announce the new volume level when the volume buttons are pressed in all apps, not just in explorer.exe.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2022.3.2
#### Windows version:
Windows 11, Version 22H2 (OS Build 22621.819)
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by mazen428** [comment]
It's meant to be that way and therefore it isn't a bug. Unsure as to why.

**Comment by mohammad-aloufi** [comment]
Behavior seemed like a bug to me since there is no reason for it to be inconsistent like that.

**Comment by josephsl** [comment]
Hi,

I and @LeonardDer had a talk about this a while back, and at least one issue was filed that requested a similar thing. This is a duplicate of #13197, and #13415 provides an explanation as to why.

Thanks.
